kolibri/utils/server.py

```diff
@@ -116,7 +119,7 @@ def start_command(cls):
 
     def run(self):
         # Try to do the vacuum every 3 hours
-        call_command("vacuumsqlite", interval=10)
+        call_command("vacuumsqlite", interval=True)
```

Comment:
can we rename this from `interval` to `scheduled`, `repeat`, or something similar?